src/gracenote.ts

```diff
@@ -46,32 +46,22 @@ export class GraceNote extends StaveNote {
       return this.stemExtensionOverride;
     }
 
-    const glyphProps = this.getGlyphProps();
-    if (glyphProps) {
-      let ret = super.getStemExtension();
-      if (glyphProps.stem) {
-        const staveNoteScale = this.getStaveNoteScale();
-        ret = (Stem.HEIGHT + ret) * staveNoteScale - Stem.HEIGHT;
-      }
-      return ret;
-    }
-
-    return 0;
+    let ret = super.getStemExtension();
+    ret = Stem.HEIGHT * this.getStaveNoteScale() - Stem.HEIGHT + ret;
+    return ret;
   }
 
   getStaveNoteScale(): number {
-    return this.renderOptions.glyphFontScale / Tables.NOTATION_FONT_SCALE;
+    return 2 / 3;
```

Comment:
I think 2/3 should be the correct target scale for grace notes. However, the final render seems smaller than 2/3.

Reply:
Hi Ron, please ignore this one. As I said below, this is corrected once vf5/note is fully integarted  and I do not feel like looking for a hack for the intermediate step. I will set an issue.